src/java.desktop/share/classes/javax/swing/JFileChooser.java

```diff
@@ -532,7 +533,7 @@ public void setSelectedFile(File file) {
      */
     @BeanProperty(description
             = "The list of selected files if the chooser is in multiple selection mode.")
-    public void setSelectedFiles(File[] selectedFiles) {
+    public void setSelectedFiles(@Nullable File[] selectedFiles) {
```

Comment:
Here, too, `@Nullable` should be on the array type.

Reply:
Here the array itself can be null, so I've pushed a fix for this. 
The elements of the array "may" be null, but probably shouldn't be null.